Sources/Frisbee/Interactors/Cancellable.swift

```diff
@@ -0,0 +1,7 @@
+public protocol Cancellable {
```

Comment:
To wrapping direct Foundation stuffs it is used Adapter pattern. It is exactly what you do, but in Frisbee we are using the Adapters folder and Adapter suffix. Could you update?

Reply:
Would you call CancellableAdapter, URLSessionTaskAdapter, SessionTaskAdapter or something else?  I didn't see any adapter the Foundation class could just adopt so I wonder if I need to change something besides the name.

Reply:
Just the names and folder location. The protocol will be `CancellableAdapter` and the implementation with Foudantation library will be `URLSessionTaskAdapter`.